+nix/Section.m

```diff
@@ -87,6 +87,26 @@
         % Property methods
         % ----------------
 
+        %-- As "datatype" provide one of the nix.DataTypes. Alternatively
+        %-- a string stating one of the datatypes supported by nix can be provided.
+        function p = create_property_data_type(obj, name, datatype)
+            p = nix.Property(nix_mx('Section::createPropertyDataType', ...
```

Comment:
I'd make it just

create_property(obj, name, datatype)

and the other one 

create_property_with_values

such that at least one is short)